src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/GenerateCoverSheetService.java

```diff
@@ -54,38 +56,43 @@ public CaseDocument generateApplicantCoverSheet(final CaseDetails caseDetails, f
             config.getBulkPrintFileName());
     }
 
-    private void prepareApplicantCoverSheet(CaseDetails caseDetails) {
+    void prepareApplicantCoverSheet(CaseDetails caseDetails) {
         BulkPrintCoverSheet.BulkPrintCoverSheetBuilder bulkPrintCoverSheetBuilder = BulkPrintCoverSheet.builder()
             .ccdNumber(caseDetails.getId().toString())
             .recipientName(join(nullToEmpty(caseDetails.getData().get("applicantFMName")), " ",
                 nullToEmpty(caseDetails.getData().get("applicantLName"))));
 
-        Object respondentAddress = caseDetails.getData().get("applicantAddress");
-        Object solicitorAddress = caseDetails.getData().get("solicitorAddress");
+        Map applicantAddress = (Map) caseDetails.getData().get("applicantAddress");
```

Comment:
Just to check. What happens to this field when running 'update contact details' and leaving it empty? Would it still be a parent object with children (i.e. addressLine1, etc) that are empty?
If yes, then casting to Map I think is okay, but if it sets it to an empty string (i.e. "") then this would return a class cast exception :)

Reply:
Discussed on Slack. Checked on AAT, it will be set to null and so ObjectUtils.isNotEmpty() will cover the null value.